tests/spatial/translation_test.py

```diff
@@ -45,3 +45,13 @@ def test_can_set_value_of_0d_variable():
     var = translation(value=value)
     var.value += value
     assert np.array_equal(var.value, value + value)
+
+
+def test_inverse():
+    rng = np.random.default_rng()
+    value = rng.random((3, ))
+    transform = translation(value=value)
```

Comment:
Might be worth adding units to vectors and transforms in the cases where it is possible?